docs/quick-start.md

```diff
@@ -0,0 +1,97 @@
+# Quick Start Guide
+
+## Overview
+
+This guide is designed to get you up and running working with the Jetpack Monorepo quickly following recommended and supported guidelines.
+
+This guide assumes you are using MacOS or a Linux machine and are an Automattician. For more detailed information, including alternate local dev environments, running unit tests, best coding practices, and more, you can use the [full Development Environment guide here](development-environment.md#clone-the-repository). 
+
+## Installation
+
+Prior to installation, we recommend using [`Homebrew`](https://brew.sh/) to manage installations and [`nvm`](https://github.com/nvm-sh/nvm/) to manage Node.js versions. If you don't already have those installed, you can do so by copy/pasting each of the following commands and running them in your terminal:
+
+- Homebrew: `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+- nvm: `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash`
+
+The Jetpack Monorepo requires the following to be installed on your machine:
+
+- Start by cloning the GitHub repo:
+	- `git clone https://github.com/Automattic/jetpack.git`
+	- If you're not an Automattician, you can [fork the repo following the instructions here](https://docs.github.com/en/get-started/quickstart/contributing-to-projects).
+- Bash (will need to be updated from default Mac version): `brew install bash`
+- jq (JSON processor used in scripts): `brew install jq` 
+- Node.js (used for build process and our CLI): `nvm install && nvm use`
+- PNPM (a Node.js package manager): `npm install -g pnpm`
+- Composer (our PHP package manager): `brew install composer`
+- PHP (the language at the core of the WordPress ecosystem): `source .github/versions.sh && brew install php@$PHP_VERSION`
```

Comment:
```suggestion
- PHP (the language at the core of the WordPress ecosystem): `source .github/versions.sh && brew install php@$PHP_VERSION`
- Composer (our PHP package manager): `brew install composer`
```

For me, installing Composer first installed PHP, but not the version we were wanting (8.1 instead). Then, I would have to install PHP per the version set in `versions.sh`. Flipping this order should speed things up for those on a sparkingly clean system.

I'll be back to test the rest of the script. The VM I setup to run through this on a clean system ran out of space, so going to lunch it up...